.github/workflows/publish-casper-client-sdk.yml

```diff
@@ -10,25 +10,34 @@ on:
 jobs:
   publish:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
 
     strategy:
       matrix:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-        node-version: [16.x]
+        node-version: [20.x]
 
     steps:
     - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b #v3.0.2
+    - name: Set release tag
+      run: |
+        if [ ${{ github.event.release.target_commitish }} == 'condor-rc' ]; then
+          echo "release_tag=condor-rc" >> $GITHUB_ENV
+        else
+          echo "release_tag=latest" >> $GITHUB_ENV
+        fi
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@5b949b50c3461bbcd5a540b150c368278160234a #v3.4.0
       with:
         node-version: ${{ matrix.node-version }}
         registry-url: 'https://registry.npmjs.org'
     - run: npm install
-    - name: Publish to NPM
-      run: npm publish --access public
-      env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+    - uses: JS-DevTools/npm-publish@v1
```

Comment:
NIT: Could we upgrade this script to the latest following this migration [guide](https://github.com/JS-DevTools/npm-publish#v1-to-v3)? or use github commit instead of version tag.